benchmarks/scripts/compare.sh

```diff
@@ -17,11 +17,10 @@ if [[ "$?" -ne 0 ]]; then
     exit 1
 fi
 
-# Checking that s3cmd is installed
-command -v s3cmd > /dev/null 2>&1
+# Checking that wget is installed
+command -v wget > /dev/null 2>&1
```

Comment:
wget is not always there, it would be preferable to use `curl -O` instead.

Reply:
You're right; I should've done that in my PR. Instead, I provided the necessary edits to rely on `curl` instead of `wget` :+1: